### PR TITLE
bootcamp ecommerce

### PR DESCRIPTION
**Pull request by abeglova**
## Description
closes https://github.com/mitodl/ol-data-platform/issues/540

## Motivation and Context
Adds
int__bootcamps__ecommerce_wiretransferreceipt
int__bootcamps__ecommerce_receipt
int__bootcamps__order

stg__bootcamps__app__postgres__ecommerce_wiretransferreceipt
stg__bootcamps__app__postgres__ecommerce_receipt
stg__bootcamps__app__postgres__ecommerce_line
stg__bootcamps__app__postgres__ecommerce_orderaudit
stg__bootcamps__app__postgres__ecommerce_order

## How Has This Been Tested?
dbt run
dbt test

### CODE REVIEW

File: src/ol_dbt/models/staging/bootcamps/_stg_bootcamps__models.yml [hunk 1]
```diff
@@ -2,6 +2,195 @@
 version: 2
 
 models:
+- name: stg__bootcamps__app__postgres__ecommerce_wiretransferreceipt
```
Comment (rachellougee) [inline src/ol_dbt/models/staging/bootcamps/_stg_bootcamps__models.yml:5]: Can we add tests for unique key and not null in this model as we did for other staging models?

File: src/ol_dbt/models/staging/bootcamps/_stg_bootcamps__models.yml [hunk 2]
```diff
@@ -2,6 +2,195 @@
...
+  - name: courserun_id
+    description: int, foreign key in courses_courserun of the purchased run
+    tests:
+    - dbt_expectations.expect_column_to_exist
+    - not_null
+  tests:
+  - dbt_expectations.expect_table_column_count_to_equal:
+      value: 7
+- name: stg__bootcamps__app__postgres__ecommerce_orderaudit
+  columns:
```
Comment (rachellougee) [inline src/ol_dbt/models/staging/bootcamps/_stg_bootcamps__models.yml:112]: Can we add tests to indicate which one is unique for this model as well?

**Review by rachellougee — APPROVED** [review]
small comments, otherwise LGTM

**Comment by rachellougee** [comment]
@abeglova Could you fix the tests for this PR? I've got some test errors related to intermediate models merged from this PR. e.g. int__bootcamps__order
